test/functionalTest/cases/0000_ipv6_support/ipv6_only.test

```diff
@@ -155,4 +155,4 @@ Date: REGEX(.*)
 brokerStop CB
 accumulatorStop ${LISTENER_PORT}
 accumulatorStop ${LISTENER2_PORT}
-#dbDrop CB
```

Comment:
I did a general look for `#dbDrop CB` while I was testing and discover this old one. I have take the opportunity to fix it.

NTC (only informative)